### DISCUSSION TITLE
Putting false positives in their own "stranger" bucket

### DISCUSSION THREAD

**Discussion by ralphv** [exadel-inc/CompreFace#1228] [Q&A]
When I get some false positives where a person is identified as person X but he/she is not person X. How can I tell the model that it should be unknown and not person X.

Was thinking of creating a stranger person and just train all those unknows/people I don't care about into that bucket, but I wonder if doing that would affect the accuracy of the other sets and get more errors.

What's a good approach here to telling the model this is not person X?

Thank you.

**Comment by rafox2005** [comment]
Hi Ralph, what worked here was that anytime the similarity is under 50 (or other threshold defined according to the use case) we create a new person using an UUID and then later we manually assign a name to the UUID if we know the person. You can assign an unknown label also using your application.

If it is a real false positive (like a wrong person with 99 similarity), then we manually exclude the example from the database.
